app/controllers/bills_controller.rb

```diff
@@ -0,0 +1,43 @@
+class BillsController < ApplicationController
```

Comment:
no ability checking?

Reply:
how can I check the ability for controller only not (model)?